src/lib.rs

```diff
@@ -1432,6 +1434,47 @@ impl<A, S, D> ArrayBase<S, D> where S: Data<Elem=A>, D: Dimension
         iterators::new_axis_iter_mut(self.view_mut(), axis)
     }
 
+    /// Return an iterator that traverses over `axis` by chunks of `size`,
+    /// yielding non-overlapping subviews along that axis.
+    ///
+    /// Iterator element is `ArrayView<A, D>`
+    ///
+    /// **Panics** if `axis` is out of bounds.
+    ///
+    /// ```
+    /// use ndarray::Array;
+    ///
+    /// let a = Array::from_iter(0..28).reshape((2, 7, 2));
+    /// let mut iter = a.axis_chunks_iter(1, 2);
+    /// assert_eq!(2, iter.next().unwrap().shape()[1]);
+    /// assert_eq!(2, iter.next().unwrap().shape()[1]);
+    /// assert_eq!(2, iter.next().unwrap().shape()[1]);
+    /// assert_eq!(1, iter.next().unwrap().shape()[1]);
```

Comment:
This example is unfortunately too cryptic. We can show just the first view in the iterator instead. Or some pretty ascii pictures... (I can add that if nothing else comes up)